src/pages/ReimbursementAccount/NonUSD/SignerInfo/HangTight.tsx

```diff
@@ -16,7 +16,7 @@ function HangTight({tempSubmit}: {tempSubmit: () => void}) {
     const {paddingBottom: safeAreaInsetPaddingBottom} = useSafeAreaPaddings();
 
     const handleSendReminder = () => {
-        // TODO remove that
+        // TODO this should send a message to the email provided in the previous step
```

Comment:
What was this for?

Reply:
It is a handler for the button on the last screen below, on the right:

![Screenshot 2025-04-15 at 10 15 22](https://github.com/user-attachments/assets/5bbfca3d-871c-479e-b518-5e9a74b7780b)

I presumed I will call the `AskForCorpaySignerInformation` endpoint, is it correct? And this is part of another issue, will be done in separate PR after we merge this one.

Reply:
Ah ok yeah, sounds good